OCPBUGS-56039: Update supported languages in Internationalization docs (#15046)
openshift/console: INTERNATIONALIZATION.md

#### Translations

OpenShift is currently translated into five languages: Chinese (Simplified), French, Japanese, Korean, and Spanish.

> The Console translations rely on collaboration with the Red Hat Globalization team. The workflow involves exporting console strings from the public and packages folders to the Phrase TMS portal for translations. The completed translations are then integrated into the console codebase when they are ready.

Reviewer: The first sentence is a little confusing to me. Is that the console team collaborating with the RH Globalization team? Maybe something like this:

```suggestion
Translations in the Console are done in collaboration with the Red Hat Globalization team. The workflow involves exporting console strings from the public and packages folders to the Phrase TMS portal for translations. The completed translations are integrated into the console codebase when they are ready.
```
